helsingborg-stad/app-mitt-helsingborg
# Feat/remove pin info modal

## Explain the changes you’ve made
Remove PinInfoModal component since its not going to be used

## Explain why these changes are made
Removed component and path to storybook

## Explain your solution
All files deleted 

## How to test

1. Checkout this branch
3. Fire upp the simulator by running the command `yarn ios`
4. The application should run as before

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
